torch/nn/quantized/modules/linear.py

```diff
@@ -4,6 +4,8 @@
 from ...modules.module import Module
 from ...modules.linear import Linear as NNLinear
 
+from typing import Optional
```

Comment:
Does this work with Python 2?

Reply:
It works as a backport library https://pypi.org/project/typing/

We use it in test_jit